formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -306,7 +309,6 @@ let myFun
 
 let myFun
     (X (hello [@onHello]) | Y (hello [@onHello])) => hello;
-
 /* Another bug: Cannot have an attribute on or pattern
-   let myFun = fun ((X hello | Y hello) [@onOrPattern]) => hello;
-   */
+let myFun = fun ((X hello | Y hello) [@onOrPattern]) => hello;
```

Comment:
I had to put a lot of effort specifically into *not* maintaining original formatting. The reason is that when you reindent a large file, the original number of leading spaces needs to change to reflect the indentation.